src/reducers/properties.js

```diff
@@ -37,4 +41,24 @@ export function getDashboardNamespace(state) {
   return state.DashboardNamespace;
 }
 
+export function getDashboardVersion(state) {
+  return state.DashboardVersion;
+}
+
+export function getPipelineNamespace(state) {
+  return state.PipelineNamespace;
+}
+
+export function getPipelineVersion(state) {
+  return state.PipelineVersion;
+}
+
+export function getTriggersNamespace(state) {
+  return state.TriggersNamespace;
+}
+
+export function getTriggersVersion(state) {
+  return state.TriggersVersion;
+}
+
```

Comment:
Since most of these are only used on the About page we could just add one new selector that returns the entire properties object.

The existing selectors (isReadOnly etc.) are mostly a convenience since those values are used in multiple places.

I'm OK with either approach, but it would be nice to see some tests for the new selectors (we've had a couple of typos in selectors that went unnoticed before due to missing tests).

Reply:
Added the missing tests